functions/Get-DbaStartupParameter.ps1

```diff
@@ -59,8 +59,17 @@ Logs in to WMI using the ad\sqladmin credential and gathers simplified informati
 			$servercount++
 			try
 			{
-				$instancename = ($servername.Split('\'))[1]
-				Write-Verbose "Attempting to connect to $servername"
+				#$servername, $instancename = ($sqlserver.Split('\'))
+				if ($null -eq $SqlServer.name)
+				{
+					$servername, $instancename = ($sqlserver.Split('\'))
```

Comment:
Isn't $sqlserver the collection?
Wouldn't it be $servername that can be split?
Or else rename the variable in the foreach ()
Actually, maybe it's the right time to switch to   $instance in $sqlinstance?